src/util/classes.ts

```diff
@@ -42,12 +47,26 @@ export const defaultCategories: Category[] = [
   { name: ['Media', 'Video'], rule: { type: 'regex', regex: 'YouTube|Plex|VLC' } },
   {
     name: ['Media', 'Social Media'],
-    rule: { type: 'regex', regex: 'reddit|Facebook|Twitter|Instagram|devRant', ignore_case: true },
+    rule: {
+      type: 'regex',
+      regex: 'reddit|Facebook|Twitter|Instagram|devRant',
+      ignore_case: true,
+    },
+  },
+  {
+    name: ['Media', 'Music'],
+    rule: {
+      type: 'regex',
+      regex: 'Spotify|Deezer|Audacity',
```

Comment:
I feel like the same discussion regarding consumption vs production applies to Audacity. Perhaps move it?
```suggestion
      regex: 'Spotify|Deezer',
```